spinnman/transceiver.py

```diff
@@ -2083,7 +2083,7 @@ def wait_for_cores_to_be_in_state(
                     time.sleep(time_between_polls)
 
         # If we haven't reached the final state, do a final full check
-        if processors_ready != len(all_core_subsets):
+        if processors_ready < len(all_core_subsets):
```

Comment:
What happens when the number of processors ready is greater than the number of cores?

Reply:
This is a bit of a horrid fix in reality, as it is fixing the issue that all_core_subsets might only be a subset of the app_id that you are querying (because of changes in FEC).  If there are more processors ready than I need, these are probably just other processors.  This needs fixing in other ways, but this is reflected in SpiNNakerManchester/SpiNNFrontEndCommon#282.

Reply:
OK. Well, not OK but whatever.